.github/workflows/publish-helm-charts.yml

```diff
@@ -30,11 +33,21 @@ jobs:
           format: "${major}.${minor}.${patch}"
           change_path: "./charts"
           bump_each_commit: true
+          namespace: helm
+
+      - name: "Remove -helm from ver number"
```

Comment:
it took me a minute to walk through what this is doing, can we get an example string in the comments like
`1.2.3-helm -> 1.2.3` or something?